### PR TITLE
add an delay， try to avoid anti-bot check

### PR DESCRIPTION
**Pull request by xelmirage**
there is some chance to trigger google's anti-bot check. I don't know in what condition it will be triggered, for starters, add some delay.

### CODE REVIEW

**Comment by naschorr** [comment]
I've been thinking about adding a random wait time onto the innermost setTimeout of main() to do exactly what you're proposing, however it's hard to know what to do for sure without knowing how the pixls dev(s?) are detecting bots. For example, if they're looking at tile placements per hour, then there isn't really a way to avoid that check other than to simply place less tiles per hour. Similarly, if they're looking at the consistency of time between placements, then any constant, or small enough random timeout based solutions will probably also be ineffective.

Basically, I don't really know, and I'd rather just prioritize getting as many tiles placed as possible, and not wasting time on potentially pointless captcha avoidance.

**Comment by xelmirage** [comment]
yeah I know, it's hard to determine the way pxls detects bots. At the worst condition it may use the way user's mouse moves on the canvas (like the one click captcha verification), which will make us simulate a mouse movement. Or the captcha just randomly pops up, because when I placed pixels manually, the captcha just pops randomly.

So maybe a pop-up nitification when a captcha shows up is a good compromise. Just alert the user to turn to the captcha as soon as possible?

**Comment by naschorr** [comment]
Yeah, I'm surprised that pxls isn't using a mouse position based solution. It seems like it'd be a pretty easy way to perform the detection reliably, but there isn't a whole lot any of us can do about that.

Notifications seem like a really good idea, especially since the user can easily configure their permissions in the browser settings without any extra work on my end. So, I've added them to the dev branch in 9d3f5ca for testing.
